packages/swirl-components/src/components/swirl-table/swirl-table.tsx

```diff
@@ -255,8 +255,7 @@ export class SwirlTable {
     const leftOffsetForStickyColumn = column.sticky
       ? this.getLeftOffsetForStickyColumn(columns, colIndex)
       : 0;
-    const columnWidth =
-      column.width || `${column.getBoundingClientRect().width}px`;
+    const columnWidth = `${column.getBoundingClientRect().width}px`;
```

Comment:
**note:** @webspaceadam Philipp gave me the hint to double check the table column width settings, which caused issues in the past. I indeed found an issue when setting `width` and `min-width`, which caused the row cell having a different width than the column.

In order to fix this mismatch, the cell width is now only based on the actual column width using `getBoundingClientRect()`